### PR TITLE
Sample tests for play-pac4j project

### PR DESCRIPTION
**Pull request by furknyavuz**
Hi @leleuj,

This pull request is sample for the unit tests that we plan to write in order to raise the overall coverage of the project to around 80%.

Please let me know if you have any questions.

Furkan Yavuz
DevFactory - Code Quality Team

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -119,6 +119,26 @@
                 <version>4.10</version>
                 <scope>test</scope>
             </dependency>
+            
+            <!-- Powermock Mocking Library -->
+			<dependency>
+			    <groupId>org.powermock</groupId>
+			    <artifactId>powermock-core</artifactId>
+			    <version>1.6.4</version>
```
Comment (leleuj) [inline pom.xml]: Could we have this version number as a property?

File: src/test/java/org/pac4j/play/PlayWebContextTest.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
+package org.pac4j.play;
```
Comment (leleuj) [inline src/test/java/org/pac4j/play/PlayWebContextTest.java]: Can you add our standard header?

File: src/test/java/org/pac4j/play/PlayWebContextTest.java [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+import org.pac4j.core.context.session.SessionStore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import play.mvc.Http.Context;
+import play.mvc.Http.Request;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({ Context.class, Request.class, PlayWebContext.class })
+public class PlayWebContextTest {
```
Comment (leleuj) [inline src/test/java/org/pac4j/play/PlayWebContextTest.java]: A javadoc comment with the `@since` and `@author` annotations would be great here.

**Comment by leleuj** [comment]
Great! Thanks. I have a few minor comments.

Is this ready for merge or do you plan to add new tests?

File: pom.xml (hunk 1 above)
Comment (furknyavuz) [inline pom.xml]: Sure,

File: src/test/java/org/pac4j/play/PlayWebContextTest.java (hunk 3 above)
Comment (furknyavuz) [inline src/test/java/org/pac4j/play/PlayWebContextTest.java]: Sure,

File: src/test/java/org/pac4j/play/PlayWebContextTest.java (hunk 2 above)
Comment (furknyavuz) [inline src/test/java/org/pac4j/play/PlayWebContextTest.java]: Sure,

**Comment by furknyavuz** [comment]
Hi @leleuj,

I gathered all your comments, I'll fix them as soon as possible.

This commit only includes sample tests. After you accept this pull request, we will make another commit includes all test cases.

**Comment by leleuj** [comment]
OK. Many thanks.

**Comment by furknyavuz** [comment]
You'r welcome @leleuj,

I made fix. Can you view it? If you are ok with it, please merge this pull request and then we can start full test coverage.

Thanks

**Comment by leleuj** [comment]
Perfect. Thanks

**Pull request merged**